### PR TITLE
DDF-2748 updated changelog

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

updates changelog for 2.10.1 release
#### Who is reviewing it?

@mcalcote @Lambeaux @jrnorth 

#### Select at least one member from relevant component team(s) from below.

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR.

@coyotesqrl
@figliold
@lessarderic
@pklinef
@shaundmorris
@stustison

#### How should this be tested?

Preview changes at https://github.com/ricklarsen/ddf/blob/DDF-2748/CHANGES.md

#### Any background context you want to provide?

Seeking additional tasks completed/notes to add. Leave as a comment on this PR for inclusion.

#### What are the relevant tickets?

[DDF-2748](https://codice.atlassian.net/browse/DDF-2748)

#### Checklist:
- [ ] Documentation Updated
- [x] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -34,15 +33,16 @@ replace with next unreleased version
...
 
 ### NOTES
 
--
+- Summary of changes requiring user action.
 
 ### NEW FEATURES
-- [DDF-2818](https://codice.atlassian.net/browse/DDF-2818) Upgrade Solr from version 6.0.0 to version 6.4.1 
+- [DDF-2818](https://codice.atlassian.net/browse/DDF-2818) Upgrade Solr from version 6.0.0 to version 6.4.1
 - [DDF-2815](https://codice.atlassian.net/browse/DDF-2815) Added UTM support for the CSW Endpoint
```
Comment (ricklarsen) [inline CHANGES.md]: added

File: CHANGES.md [hunk 2]
```diff
@@ -51,24 +51,24 @@ replace with next unreleased version
 ### BUG FIXES
 
 - List bugs fixed here.
```
Comment (ricklarsen) [inline CHANGES.md]: updated.

Comment (ricklarsen) [inline CHANGES.md]: I'm now recommending we use this new approach of using the jira-generated release notes to build the changelog.

File: CHANGES.md [hunk 3]
```diff
@@ -1,276 +1,2504 @@
...
+	</li>
+</ul>
+
+<h3>Story</h3>
+<ul>
+	<li><a href='https://codice.atlassian.net/browse/DDF-1765'>DDF-1765</a> - As a user, I want to add a layer to the Search UI map from a KML file so that I can visualize my KML data
+	</li>
+</ul>
+    
+<h3>New Feature</h3>
```
Comment (lessarderic) [inline CHANGES.md]: ❓ Shouldn't this be in the _Story_ section?

File: CHANGES.md [hunk 4]
```diff
@@ -1,276 +1,2504 @@
...
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2731'>DDF-2731</a> - Move PropertiesFileReader where it can be re-used
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2798'>DDF-2798</a> - There are several possible null pointers within the ECP handlers
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2821'>DDF-2821</a> - Check audit messages for special characters and encode them before writing the audit
+	</li>
+</ul>
+
+<h3>Spike</h3>
```
Comment (lessarderic) [inline CHANGES.md]: ❗️ Do we really need to include spikes in the change logs? Those do not (and should not) involve code changes.

File: CHANGES.md [hunk 5]
```diff
@@ -1,276 +1,2504 @@
...
-### KNOWN ISSUES
-
-- [DDF-2513](https://codice.atlassian.net/browse/DDF-2513) Password fields in Admin UI appear to be populated with a password when none has been set
-- [DDF-2507](https://codice.atlassian.net/browse/DDF-2507) CSW queries that include numeric criteria always return 0 results
-- [DDF-2506](https://codice.atlassian.net/browse/DDF-2506) Sorting by distance can take minutes on large indexes
-- [DDF-2504](https://codice.atlassian.net/browse/DDF-2504) Multiple SSO sessions are assigned per browser, which can cause unexpected logout from UI, forcing the user to log in again.
-- [DDF-2455](https://codice.atlassian.net/browse/DDF-2455) SchematronValidationService incorrectly appears as a Source in the Sources Admin UI tab
-- [DDF-2432](https://codice.atlassian.net/browse/DDF-2432) Metacard validation UI incorrectly reports a Metacard is a duplicate of itself
+	Release Date: 2017-02-08    
+<h3>Sub-task</h3>
```
Comment (lessarderic) [inline CHANGES.md]: ❗️ Don't think we should include sub-tasks in the change log.

File: CHANGES.md [hunk 6]
```diff
@@ -1,276 +1,2504 @@
...
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2120'>DDF-2120</a> - As an Integrator, I need the DDF API to provide a status of the number of results returned/success/failed/partial of the query that is in progress
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2162'>DDF-2162</a> - Add the ability to inject attributes into Metacard Types
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2170'>DDF-2170</a> - Add srcclr profile to root pom
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2196'>DDF-2196</a> - Separate admin duties via Admin Console
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2535'>DDF-2535</a> - As a user, I w
```
Comment (lessarderic) [inline CHANGES.md]: ❗️ Remove?

Comment (lessarderic) [inline CHANGES.md]: ❗️ Remove.

File: CHANGES.md (hunk 3 above)
Comment (ricklarsen) [inline CHANGES.md]: It's just pulling from the types on the tickets, and I haven't edited any issues. (yet)

File: CHANGES.md (hunk 4 above)
Comment (ricklarsen) [inline CHANGES.md]: No, I'll pull them out.

File: CHANGES.md (hunk 5 above)
Comment (ricklarsen) [inline CHANGES.md]: Will remove.

File: CHANGES.md (hunk 6 above)
Comment (ricklarsen) [inline CHANGES.md]: Will do

Comment (ricklarsen) [inline CHANGES.md]: Will do

File: CHANGES.md (hunk 3 above)
Comment (ricklarsen) [inline CHANGES.md]: updated ticket: https://codice.atlassian.net/browse/DDF-2039

File: CHANGES.md (hunk 5 above)
Comment (ricklarsen) [inline CHANGES.md]: removed

File: CHANGES.md (hunk 6 above)
Comment (ricklarsen) [inline CHANGES.md]: removed

**Comment by ricklarsen** [comment]
@lessarderic Updated with your requested changes.

**Comment by jlcsmith** [comment]
This is a nit, but I would prefer to see internationalized date formats.  So instead of of this:
Release Date: 2017-03-08 
use:
Release Date: Mar 8, 2017

**Pull request merged**